sdk/src/main/java/com/hedera/hashgraph/sdk/Executable.java

```diff
@@ -82,15 +87,30 @@ public SdkRequestT setNodeAccountIds(List<AccountId> nodeAccountIds) {
 
     @FunctionalExecutable
     public CompletableFuture<O> executeAsync(Client client) {
-        return onExecuteAsync(client).thenCompose((v) -> executeAsync(client, 1, null));
+        return onExecuteAsync(client).thenCompose((v) -> {
+            if(nodeAccountIds.isEmpty()) {
```

Comment:
Can we extract this node stuff into a separate method; something like `setNodes()`.

Reply:
We can, but it will only ever be used in one place.  I'm generally in the habit of not factoring something out until it's used in multiple places.